notebooks/242-freevc/242-freevc.ipynb

```diff
@@ -0,0 +1,811 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.    iface.launch()
</pre><p>on some systems it does not work if we do not run it in share=True, please check for example on windows laptop</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1042/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>